pkg/vtpm/src/server.cpp

```diff
@@ -101,13 +263,20 @@ int readMessage(int sock, google::protobuf::uint32 size)
             output_file.close();
         }
     }
-    size = response.ByteSize() + CODED_STRM_HDR_LEN;
-    char *resp_buffer = new char [size];
-    google::protobuf::io::ArrayOutputStream aos(resp_buffer, size);
-    CodedOutputStream *coded_output = new CodedOutputStream(&aos);
-    coded_output->WriteVarint32(response.ByteSize());
-    response.SerializeToCodedStream(coded_output);
-    send(sock, (void *)resp_buffer, size, 0);
+    //Clear all the files: inputFiles and expectedFiles
+    for (int i=0; i < request.expectedfiles_size(); i++) {
+        std::string expectedFile = request.expectedfiles(i);
+        ostringstream command;
+        command << "rm -f " << expectedFile;
+        system(command.str().c_str());
```

Comment:
Can't use system. You can call unlink() directly.
UNLINK(2)                  Linux Programmer's Manual                 UNLINK(2)

NAME
       unlink, unlinkat - delete a name and possibly the file it refers to

SYNOPSIS
       #include <unistd.h>

       int unlink(const char *pathname);

Reply:
fixed it in the latest diff.